pkg/domain/infra/abi/terminal/sigproxy_linux.go

```diff
@@ -16,6 +16,48 @@ import (
 // runc is using the same value.
 const signalBufferSize = 2048
 
+func ProxySignalsRemote(ctrID string, killFunc func() error) {
```

Comment:
I just copy paste `ProxySignals` to a new method that accepts a kill function. I can refactor this and use it also in non-remote parts as well. Just lemme know what you think @mheon